### ISSUE TITLE
Can't run fresh install on Ubuntu 17/04 x64

### ISSUE THREAD

**Issue by mrPsycho** [serge-rider/dbeaver#2170]
Hello! 

i tried to install beaver-ce on Ubuntu x64. Installation was successful, but it won't start:

```
!SESSION 2017-09-18 17:00:20.769 -----------------------------------------------
eclipse.buildId=unknown
java.version=9-Ubuntu
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US
Command-line arguments:  -os linux -ws gtk -arch x86_64

!ENTRY org.eclipse.osgi 4 0 2017-09-18 17:00:30.171
!MESSAGE Application error
!STACK 1
java.lang.NoClassDefFoundError: java/sql/SQLException
	at org.jkiss.dbeaver.core.application.DBeaverApplication.start(DBeaverApplication.java:119)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:547)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
Caused by: java.lang.ClassNotFoundException: java.sql.SQLException
	at java.base/java.lang.ClassLoader.findClass(ClassLoader.java:688)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:543)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:371)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:364)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:161)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:476)
	... 14 more
```

why?

also, i don't need mysql.

**Comment by serge-rider** [comment]
This is related to Java 9 EA. Check this ticket: https://github.com/serge-rider/dbeaver/issues/1559
Running DBeaver with command line
```
-vmargs
--add-modules=java.se.ee
```
should help

**Comment by mrPsycho** [comment]
still the same ( 

```
yn@uvid:/usr/share/dbeaver$ cat /home/yn/.eclipse/org.jkiss.dbeaver.product_4.2.1_620785679_linux_gtk_x86_64/configuration/1505744081307.log
!SESSION 2017-09-18 17:14:33.030 -----------------------------------------------
eclipse.buildId=unknown
java.version=9-Ubuntu
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US
Command-line arguments:  -os linux -ws gtk -arch x86_64

!ENTRY org.eclipse.osgi 4 0 2017-09-18 17:14:42.618
!MESSAGE Application error
!STACK 1
java.lang.NoClassDefFoundError: java/sql/SQLException
	at org.jkiss.dbeaver.core.application.DBeaverApplication.start(DBeaverApplication.java:119)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:547)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
Caused by: java.lang.ClassNotFoundException: java.sql.SQLException
	at java.base/java.lang.ClassLoader.findClass(ClassLoader.java:688)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:543)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:476)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:371)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:364)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:161)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:476)
	... 14 more
yn@uvid:/usr/share/dbeaver$ cat dbeaver.ini 
-startup
plugins/org.eclipse.equinox.launcher_1.3.201.v20161025-1711.jar
--launcher.library
plugins/org.eclipse.equinox.launcher.gtk.linux.x86_64_1.1.401.v20161122-1740
-showsplash
--launcher.GTK_version
2
-vmargs
-Xms64m
-Xmx1024m
--add-modules=java.se.ee
```


vmargs already was there, so i added only --add-modules=java.se.ee

**Comment by serge-rider** [comment]
Well, Java 9 support is so tricky..
You could try to add
```
--add-modules=java.se.ee,java.sql
```
or
```
--addmods=java.se.ee,java.sql
```
Exact config depends on Java 9 version. In fact I'd recommend to use Java 8 because Java 9 is too raw at the moment.

**Comment by mrPsycho** [comment]
`--addmods=java.se.ee,java.sql`

doesn't work at all:
```
JVM terminated. Exit code=1
/usr/bin/java
-Xms64m
-Xmx1024m
--addmods=java.se.ee,java.sql
-jar /usr/share/dbeaver//plugins/org.eclipse.equinox.launcher_1.3.201.v20161025-1711.jar
-os linux
-ws gtk
-arch x86_64
-showsplash
-launcher /usr/share/dbeaver/dbeaver
-name Dbeaver
--launcher.library /usr/share/dbeaver//plugins/org.eclipse.equinox.launcher.gtk.linux.x86_64_1.1.401.v20161122-1740/eclipse_1618.so
-startup /usr/share/dbeaver//plugins/org.eclipse.equinox.launcher_1.3.201.v20161025-1711.jar
--launcher.overrideVmargs
-exitdata 315801e
-vm /usr/bin/java
-vmargs
-Xms64m
-Xmx1024m
--addmods=java.se.ee,java.sql
-jar /usr/share/dbeaver//plugins/org.eclipse.equinox.launcher_1.3.201.v20161025-1711.jar 
```

**Comment by mrPsycho** [comment]
`--add-modules=java.se.ee,java.sql` doesn't help. 

java 8... is it possible to run beaver in java8 directly? 
i'm a tester, and i need latest java for our project...

**Comment by serge-rider** [comment]
It is possible to specify JVM for DBeaver (-vm command line parameter).
Generally this is a typical issue for Eclipse + Java 9: https://stackoverflow.com/questions/34947994/neon-how-to-run-on-jdk9

**Comment by serge-rider** [comment]
This problem will be solved in #2204
